Pipfile

```diff
@@ -47,11 +47,15 @@ google-api-python-client = "<=2.60.0"
 certifi = "<=2022.6.15.1"
 mistune = "==2.0.3"
 tomte = {version = "==0.2.4", extras = ["tox", "tests"]}
-apduboy = ">=0.5.0"
 docspec = "==2.0.2"
 docspec-python = "==2.0.2"
 hexbytes = "==0.3.0"
+apduboy = {git = "https://github.com/LedgerHQ/apduboy.git"}
```

Comment:
@angrybayblade 

https://github.com/LedgerHQ/apduboy/pull/3

This pr will hopefully be released by @onyb ;)

Which will mean we can remove this reference.

For now, we can reference by hash using this syntax'


{git = "https://github.com/LedgerHQ/apduboy.git#/d167f6204"}



@solarw can you please test if this would work for best practises?

Reply:
@8ball030 updated. commit id is good. added.

Reply:
nice nice nice